frontend/src/Ux/AboutUs.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from "react";
+
+export class AboutUs extends Component<any, any> {
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            isClicked: false
+        }
+    }
+
+    private changeState(){
+        const currentState : boolean = this.state.isClicked;
+        this.setState({isClicked: !currentState})
+    }
+    render() {
+        return (<>
+                <div className={`modal`}>
+                    <h3>Twórcy projektu</h3>
+                    <div className={`modal-inside`}>
+                        <h2>Paweł Paziewski-Kopczewski</h2>
+                        <h2>Filip Pawłowicz</h2>
+                        <button onClick={() => this.changeState()}>DZIĘKUJEMY</button>
```

Comment:
Try to avoid using lambda expressions in render function as it may cause performance issues. It's better to declare an event function in class body and bind it in constructor.

Reply:
done